runtime/dancebox/src/lib.rs

```diff
@@ -2576,10 +2584,29 @@ impl_runtime_apis! {
         }
     }
 
-    impl pallet_xcm_core_buyer_runtime_api::XCMCoreBuyerApi<Block, BlockNumber, ParaId> for Runtime {
+    impl pallet_xcm_core_buyer_runtime_api::XCMCoreBuyerApi<Block, BlockNumber, ParaId, NimbusId> for Runtime {
         fn is_core_buying_allowed(para_id: ParaId) -> Result<(), BuyingError<BlockNumber>> {
```

Comment:
Here we should pass:
- slot
- NimbusId

Reply:
We get `slot` from `Slotbeacon` in runtime. So, only passing `NimbusId` is sufficient.